### PR TITLE
Please review my code. 

### PR DESCRIPTION
**Pull request by macnick**

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by: Inyama, Uchechukwu Henry

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi macnick! you did a great job :clap:  :smiley: 
### But there are just few updates to make. 
- [ ] Use these breakpoints below:
    * For small screens with a width of up to 720px.
    * For medium screens with a width in the range of 720px - 1024px.
    *  For large screens with width above 1024px
- [ ]  Add the ```<main></main>``` and ```<footer></footer>``` tags, at the appropriate sections.
- [ ] Also, there are no stickler checks for this project, ensure there's a check and your files passes the test.

 - [ ] Finally, an observation; this project should be a **collaborative project**, but it was done **solo** looking at your commits, could you kindly explain? :smiley: 

### Implement these changes with your feedback, then request for another review.

#### Cheers Henry :smiley:

**Comment by macnick** [comment]
Hello Henry, 

Thank you for your review. I did the requested changes. 
To answer your question, my coding partner left microverse a week ago, so I remained alone to finish the project. 

Best Regards,

Nick

File: index.html [hunk 1]
```diff
@@ -4,29 +4,208 @@
...
+            and operated by IAC (NASDAQ: IAC).
+          </p>
+        </div>
+        <div class="row">
+          <div class="col-xs-7 col-md-4">
+            <div class="wpb">
+              <div class="wpb_wrapper">
+                <h2><span style="color: #cf0000;">Culture</span></h2>
+              </div>
+            </div>
```
Comment (samgaco) [inline index.html:74]: [REQUIERED] You have used `<div>` tags within all the project but you could include the use of [HTML semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp) like the `<section>` tag.

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @macnick :hand: 

#### As always, your project looks perfect :100: 

Anyways, We have to review some minor things before moving on:

1.  > Also, there are no stickler checks for this project, ensure there are a check and your files pass the test.

You have more information about this point [here](https://github.com/microverseinc/linters-config/tree/master/css).

2. You should incorporate [HTML semantic elements](https://internetingishard.com/html-and-css/semantic-html/) in your project.


### So, please just work on these 2 :crossed_fingers: points and submit back for another code review. Regards!

* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 


* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Comment by macnick** [comment]
Hello @samgaco,

**Thank you** for your helpful review and kind words. It is a pleasure working with you.

Stickler added and semantics are fixed. 

I didn't exactly follow the Github Flow on this one since they told us on HTML & CSS projects to consider the whole project as one feature. I will take care of it on the next one though. 

Best Regards, 

Nick

**Review by offorma — APPROVED** [review]
Hi @macnick,
Well done you have completed all the requested changes and have met all the requirements for this project.:clap::clap:
This project is now complete and you can go ahead and merge to master. Good luck with your next project

**Comment by macnick** [comment]
Thank you so much, @uche-inyama, @samgaco and @offorma 

for your quick review  and clear instructions.

Best Regards, 

Nick

**Pull request merged**